backend/vcdat/FlaskTests.py

```diff
@@ -0,0 +1,55 @@
+from app import app
+import unittest
+
+
+class FlaskTests(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        pass
+
+    @classmethod
+    def tearDownClass(cls):
+        pass
+
+    def setUp(self):
+        self.app = app.test_client()
+        self.app.testing = True
+
+    def tearDown(self):
+        pass
+
+    # Status Code Tests
+    def test_home_status_code(self):
+        result = self.app.get('/')
+        self.assertEqual(result.status_code, 200)
+
+    def test_templates_status_code(self):
+        result = self.app.get('/getTemplates')
+        self.assertEqual(result.status_code, 200)
+
+    def test_graphics_methods_status_code(self):
+        result = self.app.get('/getGraphicsMethods')
+        self.assertEqual(result.status_code, 200)
+
+    def test_file_tree_status_code(self):
+        result = self.app.get('/getInitialFileTree')
+        self.assertEqual(result.status_code, 200)
+
+    def test_browse_files_status_code(self):
+        result = self.app.get('/browseFiles', query_string='path=/Users/')
```

Comment:
This is a bad path choice, since it only exists on macs. I would use the directory that this file is in, since you know that exists.